sklearn/__init__.py

```diff
@@ -16,10 +16,12 @@
 import re
 import warnings
 
+from .exceptions import SklearnDeprecationWarning
 
 # Make sure that DeprecationWarning within this package always gets printed
 warnings.filterwarnings('always', category=DeprecationWarning,
```

Comment:
There are many other DeprecationWarnings issued by scikit-learn (not for deprecated modules). I haven't switched them all to SklearnDeprecationWarning (should I?). So showing DeprecationWarning from scikit-learn is still supported, it is not required to use SklearnDeprecationWarning to make it visible, unless it is a module deprecation with stacklevel=2.